otherlibs/runtime_events/runtime_events.mli

```diff
@@ -87,6 +87,16 @@ Live blocks of a Domain's major heap pools.
 (**
 Live blocks of a Domain's major heap large allocations.
 @since 5.1 *)
+| EV_C_MAJOR_HEAP_WORDS
+| EV_C_MAJOR_ALLOCATED_WORDS
```

Comment:
The constructors above all have careful documentation. Shouldn't we try to document those as well, or at least make it explicit why they are not documented?

Reply:
Good point. Let me try to document these.

Reply:
That would generate useful documentation on the [Runtime_events module](https://ocaml.org/manual/5.2/api/Runtime_events.html). 
Unfortunately some of the other constructors are undocumented.